docs/user/supported_devices.rst

```diff
@@ -180,6 +180,7 @@ ipq40xx-generic
 * AVM
 
   - FRITZ!Box 4040 [#avmflash]_
+  - FRITZ!Box 7520 [#eva_ramboot]_
```

Comment:
See my first Comment in this PR - add a `v1 / VRX518` suffix here.

Reply:
added